c2corg_ui/templates/base.html

```diff
@@ -115,9 +133,7 @@
          module.constant('apiUrl', '${api_url}');
          module.constant('authUrl', '${request.route_url('auth')}');
          module.constant('langs', ['${"','".join(default_langs) |n}']);
-         module.config(function(vcRecaptchaServiceProvider) {
-           vcRecaptchaServiceProvider.setSiteKey('${settings["recaptcha_site_key"]}');
-         });
```

Comment:
I didn't touch this part, probably a rebase that removed it. Should it stay or should it go? :notes: